gclient/src/api/listener/mod.rs

```diff
@@ -263,4 +263,9 @@ pub trait EventProcessor {
             .flatten()
             .collect())
     }
+
+    async fn queue_processing_reverted(&mut self) -> Result<()> {
```

Comment:
why not just boolean one?

Reply:
cause error can occur on other levels

Reply:
Occurred errors do not give 100% confidence about queue processing being stopped. For example, an error we get while processing events stream is `jsonrpsee::Error::ParseError`, which tells nothing about node's queue.

Reply:
Yeah, sorry for my explanation, I'm about returning `Result<bool>`

Reply:
discussed